libs/challenge-registry/user-profile/src/lib/user-profile-starred/user-profile-starred.component.html

```diff
@@ -1,32 +1,40 @@
-<main class="main-section-stars-container">
-  <section id="main" class="main-section-stars-main-section">
-    <span class="main-section-stars-text">Starred</span>
-    <div class="main-section-stars-tab-group">
-      <button class="tab-item" 
-          (click)="clickEvent('challenges')"
-          [ngClass]=" {'active-tab': activeTab===tabs['challenges']}">
-        Challenges</button>
-      <button class="tab-item" 
-          (click)="clickEvent('organizations')"
-          [ngClass]=" {'active-tab': activeTab===tabs['organizations']}">
-        Organizations</button>
+<main>
+  <div id="stars">
+    <h3>Starred</h3>
+    <div class="tab-nav-group">
+      <button
+        class="tab-item"
+        (click)="clickEvent('challenges')"
+        [ngClass]="{ 'active-tab': activeTab === tabs['challenges'] }"
+      >
+        Challenges
+      </button>
+      <button
+        class="tab-item"
+        (click)="clickEvent('organizations')"
+        [ngClass]="{ 'active-tab': activeTab === tabs['organizations'] }"
+      >
+        Organizations
+      </button>
     </div>
-    <ng-container [ngSwitch]="activeTab">
-      <div *ngSwitchCase="tabs['challenges']" class="main-section-stars-card-group">
-        <challenge-registry-challenge-card
-          *ngFor="let challenge of challenges"
-          [challenge]="challenge"
-        >
-        </challenge-registry-challenge-card>
-      </div>
-      <div *ngSwitchCase="tabs['organizations']" class="main-section-stars-card-group">
-        <challenge-registry-organization-card
-          *ngFor="let organization of organizations"
-          [organization]="organization"
-        >
-        </challenge-registry-organization-card>
-      </div>
-    </ng-container>
-    <div class="main-section-stars-paginator"></div>
-  </section>
+    <div class="card-group">
+      <ng-container [ngSwitch]="activeTab">
+        <div *ngSwitchCase="tabs['challenges']" class="main-section-stars-card-group">
```

Comment:
@vpchung  challenge card group is actually this selector, `main-section-stars-card-group`. You are applying the flex to its parent `card-group`. Fixing this should make the challenge cards responsive :)

Reply:
Ah, good eye! Thanks, @rrchai :-)